packages/medusa/src/api-v2/admin/fulfillments/validators.ts

```diff
@@ -1,44 +1,39 @@
 import { z } from "zod"
+import { AddressPayload } from "../../utils/common-validators"
 import { createSelectParams } from "../../utils/validators"
 
 export const AdminFulfillmentParams = createSelectParams()
 
+const AdminCreateFulfillmentItem = z.object({
+  title: z.string(),
+  sku: z.string(),
+  quantity: z.number(),
+  barcode: z.string(),
+  line_item_id: z.string().optional(),
+  inventory_item_id: z.string().optional(),
+})
+
+const AdminCreateFulfillmentLabel = z.object({
+  tracking_number: z.string(),
+  tracking_url: z.string(),
+  label_url: z.string(),
+})
+
 export type AdminCancelFulfillmentType = z.infer<typeof AdminCancelFulfillment>
 export const AdminCancelFulfillment = z.object({})
 
 export type AdminCreateFulfillmentType = z.infer<typeof AdminCreateFulfillment>
 export const AdminCreateFulfillment = z.object({
```

Comment:
comment: This will need to accept `order_id` and then `order` should probably be changed to context or similar. Both are things we can add in follow up PRs, so only adding these to have them on record.

Reply:
created a ticket for it